retinanet/cfg/config.py

```diff
@@ -48,10 +43,17 @@ def _update(d, u):
         return _update(d, u)
 
     def override(self, config: Text) -> EasyDict:
-        assert isinstance(config, str), "wrong type for hparams"
```

Comment:
assert isinstance(config, str), f"`config` must be a json string but got {type(config)}"

Reply:
?

Reply:
the below line no. 47 statement can be a oneliner using above.

Reply:
Using `AssertionError` instead of `assert` is present as one of the checks for "DeepSource" bot. For sake consistency, this should be kept in the present format, If require I'll be changing it across the codebase. Also, I have tried to avoid f string for compatibility issue with python3.5

Reply:
@srihari-humbarwadi oh, ok then will use .format for compatibility purpose